src/test/java/io/reactivex/internal/operators/observable/ObservableToXTest.java

```diff
@@ -48,4 +49,31 @@ public void toFlowableLatest() {
         ts
         .assertResult(5);
     }
+
+    @Test
+    public void toFlowableError1() {
+        TestSubscriber<Integer> ts = Observable.range(1, 5)
+        .toFlowable(BackpressureStrategy.ERROR)
+        .test(1)
+        .assertError(MissingBackpressureException.class);
```

Comment:
Why not `assertFailure(MissingBackpressureException.class, 1)` to make sure `1` got through.